tipg/factory.py

```diff
@@ -1364,3 +1390,220 @@ async def tilematrixset(
             OGC Specification: http://docs.opengeospatial.org/per/19-069.html#_tilematrixset
             """
             return self.supported_tms.get(tileMatrixSetId)
+
+
+@dataclass
+class Endpoints(EndpointsFactory):
+    """OGC Features and Tiles Endpoints Factory."""
+
+    title: str = "TiPg: OGC Features and Tiles API"
+
+    # OGC Tiles dependency
+    supported_tms: TileMatrixSets = default_tms
+
+    ogc_features: OGCFeaturesFactory = field(init=False)
+    ogc_tiles: OGCTilesFactory = field(init=False)
```

Comment:
This factory combines both `OGCFeaturesFactory` and `OGCTilesFactory` factories.